doc/man3/SSL_get_version.pod

```diff
@@ -57,7 +56,7 @@ The connection uses the TLSv1.3 protocol.
 
 =item unknown
 
-This indicates that no version has been set (no connection established).
+This indicates that the protocol is in an unknown state.
```

Comment:
This does not actually look like an improvement. This sounds like: The code is really horrible and we can't even figure out in what state it is, so we just return unknown.

Reply:
I think this is a "should not happen" condition. But the case is there in the code so I think we should document it. Are you asking for a change to this PR?

Reply:
Maybe it can return it for DTLS?

Reply:
Can you just call it an unknown version?

Reply:
> Maybe it can return it for DTLS?

It shouldn't - the function supports all DTLS protocol versions.

> Can you just call it an unknown version?

Ok. I did that.